GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -51,6 +51,27 @@ + (NSString *)archivePath {
   return archivePath;
 }
 
++ (NSString *)libraryDataPath {
```

Comment:
nit: Since the method has a side effect (may create a directory) it may be useful to add mention about it in the method name to improve readability, e.g. `libraryDataPathCreateIfNotExists`. I don't have a strong opinion here, so leave it up to you.

Reply:
You're right, but the way it is is more consistent with other similar folder functions in GDT.